core/src/main/kotlin/configuration.kt

```diff
@@ -3,6 +3,25 @@ package org.jetbrains.dokka
 import java.io.File
 import java.net.URL
 
+object DokkaDefaults {
```

Comment:
Probable questions:

____ 
_Why not splitting defaults for `DokkaConfiguration`, `PassConfiguration` and `packageConfiguration` into its own defaults?_

That was what I actually did first, because it seems intuitive. However, this leads to potential inconsistencies, minor maintenance overhead and overall confusion inside the `DokkaMojo` class. 

____

_Why not calling it Dokka**Configuration**Defaults or putting DokkaConfiguration.Defaults_

This is to avoid the potential misconception that this Defaults are just used for types implementing `DokkaConfiguration`